libs/src/services/ethContracts/governanceClient.js

```diff
@@ -16,8 +38,20 @@ class GovernanceClient extends ContractClient {
     this.isDebug = isDebug
   }
 
+  getSignatureAndCallData (methodName, contractMethod) {
+    const web3 = this.web3Manager.getWeb3()
+
+    const argumentTypes = contractMethod._method.inputs.map(i => i.type)
+    const argumentValues = contractMethod.arguments
+
+    const signature = createMethodSignature(methodName, argumentTypes)
+    const callData = abiEncode(web3, argumentTypes, argumentValues)
+
+    return { signature, callData }
+  }
+
   async guardianExecuteTransaction (
-    contractRegistryKey,
+    registryKey,
```

Comment:
i think keeping it as `contractRegistryKey` is a bit more explicit too

Reply:
thanks sry, rebase issue. ty!